Kernel/Formula.cpp

```diff
@@ -776,6 +776,10 @@ Formula* Formula::createLet(unsigned predicate, VList* variables, Formula* body,
 
 Formula* Formula::quantify(Formula* f)
 {
+
+  DHMap<unsigned,TermList> tMap;
+  SortHelper::collectVariableSorts(f,tMap);
+
```

Comment:
Minor optimisation: are they keys of `tMap` not also the members of `vars` below? i.e. could we reuse `tMap` and remove `vars` altogether?

Reply:
Done